layers/+completion/unite/packages.vim

```diff
@@ -3,6 +3,6 @@
 
 MP 'Shougo/unite.vim'
 
-MP 'ctrlpvim/ctrlp.vim',      { 'on': ['CtrlP', 'CtrlPMRU'] }
+MP 'ctrlpvim/ctrlp.vim'
 
-MP 'FelikZ/ctrlp-py-matcher', { 'on': ['CtrlP', 'CtrlPMRU'] }
+MP 'FelikZ/ctrlp-py-matcher'
```

Comment:
There are many ctrlp extension and unite extension. I think this is not the best solution.